api/build.gradle

```diff
@@ -10,6 +10,6 @@ dependencies {
 }
 
 javadoc {
-    exclude 'io/opencensus/internal/**'
-    exclude 'io/opencensus/trace/internal/**'
+    exclude 'io/openconsensus/internal/**'
+    exclude 'io/openconsensus/trace/internal/**'
```

Comment:
it is not io.openconsensus just openconsensus

Reply:
Removed all.